kiwi_post_run

```diff
@@ -91,13 +93,22 @@ echo "name $NAME"
 echo "version $VERSION"
 echo "release $RELEASE"
 echo "source $IMAGE"
+echo "metadata $METADATA"
+
+# Generate metada file for container-feeder
+cat << EOF > $METADATA
+Name: $CONTAINER_RAW
+Tag: $CONTAINER_TAG
+File: $IMAGE
+EOF
```

Comment:
@flavio : Problem is that I'm not sure if we can provide to KIWI config.xml multi tags. I have to ask @davidcassany. Alle metadata comes from KIWI config.xml file